### PR TITLE
nebula.0.2.1 - via opam-publish

### PR DESCRIPTION
**Pull request by hakuch**
DCPU-16 emulator.

Nebula is a complete DCPU-16 emulator including simulated hardware devices.

---
* Homepage: https://github.com:hakuch/Nebula
* Source repo: git://github.com/hakuch/Nebula.git
* Bug tracker: https://github.com:hakuch/Nebula

---

Pull-request generated by opam-publish v0.3.0

### CODE REVIEW

File: packages/nebula/nebula.0.2.1/opam [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+maintainer: "Jesse Haber-Kucharsky <jesse@haberkucharsky.com>"
+authors: "Jesse Haber-Kucharsky <jesse@haberkucharsky.com>"
+homepage: "https://github.com/hakuch/Nebula"
+bug-reports: "https://github.com/hakuch/Nebula"
+license: "Apache 2.0"
+dev-repo: "git@github.com:hakuch/Nebula.git"
+build: [make]
+build-doc: [make "doc"]
+depends: [
+  "cmdliner" {= "0.9.7"}
```
Comment (avsm) [inline packages/nebula/nebula.0.2.1/opam]: Would it be possible to make these minimum constraints and not exact ones?  We dont put upper bounds unless there is a known incompatibility.  Thanks!

Comment (hakuch) [inline packages/nebula/nebula.0.2.1/opam]: @avsm: Absolutely. Thanks for the feedback.

**Comment by hakuch** [comment]
I fixed the `dev-repo` link. :-)

**Comment by avsm** [comment]
thanks for all the fixes!

**Comment by avsm** [comment]
Just noticed that the constraints have gone back to being `=` instead of `>=` -- did you do a force push that reverted it?

**Comment by hakuch** [comment]
Oh no! What do you suggest as the best way to fix?

**Pull request merged**